src/components/NotesMindMapView/NotesMindMapViewSagas.js

```diff
@@ -121,6 +123,12 @@ function deleteNote({ data: { note, visNetwork } }) {
     });
 }
 
+function* changeParentNote(data) 
+{
+  const response = yield noteStorage.move(data);
+  yield put({ type: "SET_CHANGE_PARENT_MODE_ACTION", data: false });
```

Comment:
Please conform to the Redux styleguide
>Model Actions as Events, Not Setters
https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters
For example:
```
  yield put({ type: "MOVE_NOTE_SUCCESS", data: { note });
```

Secondly, store the action name in NotesMindMapViewActions.js and use a variable here.

Also, please handle http request fail.